documentation/install_from_packages.markdown

```diff
@@ -1,16 +1,25 @@
 ---
 layout: default
-title: "Puppet Server: Installing From Packages"
+title: "Install Puppet Server"
 canonical: "/puppetserver/latest/install_from_packages.html"
 ---
 
 [repodocs]: https://puppet.com/docs/puppet/latest/puppet_platform.html
 
-Puppet Server is configured to use 2 GB of RAM by default. If you're just testing an installation on a Virtual Machine, this much memory is not necessary. To change the memory allocation, see [Memory Allocation](#memory-allocation).
+Puppet provides official packages that install Puppet Server 6.0 and all of its prerequisites on x86_64 architectures for the following platforms, as part of [Puppet Platform][repodocs].
 
-> If you're also using PuppetDB, check its [requirements](https://puppet.com/docs/puppetdb/latest/index.html#system-requirements).
+* Red Hat Enterprise Linux 6, 7
+* Debian 8 (Jessie), 9 (Stretch), 10 (Buster)
+* Ubuntu 18.04 (Bionic) - enable the [universe repository](https://help.ubuntu.com/community/Repositories/Ubuntu), which contains packages necessary for Puppet Server, 16.04 (Xenial)
+* SLES 12 SP1
 
-## Java support
+> Note: Java 8 runtime packages do not exist in the standard repositories for Debian 8 (Jessie).  To install Puppet Server on Jessie, [configure the `jessie-backports` repository](https://backports.debian.org/Instructions/). 
+
+### Before you begin
+
+Puppet Server is configured to use 2 GB of RAM by default. If you're just testing an installation on a Virtual Machine, this much memory is not necessary. To change the memory allocation, see [Memory Allocation](#memory-allocation).
+
+#### Java support
```

Comment:
Maybe this information be moved elsewhere?

Reply:
I think it's useful to have here, since you need to manually install java a lot of the time along with the server package.